labellab-flask/api/controllers/modelscontroller.py

```diff
@@ -0,0 +1,81 @@
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import jwt_required
+
+from api.models.Model import Model, optional_params
+from api.models.Projects import Project
+
+class Save(MethodView):
```

Comment:
Hey @laksh22 could you specify the model name more distinguishably?

Reply:
Would something like `MLModel` instead of `Model` be better?

Reply:
Yes sounds good!